tests/test_all_fields.py

```diff
@@ -171,7 +164,6 @@
         'discount', # BUG_12478 | missing subfields
         'plans', # BUG_12478 | missing subfields
         'finalized_at', # BUG_13711 | schema missing datetime format
```

Comment:
@namrata270998 please remove it as it is already added in FIELDS_TO_NOT_CHECK variable.
https://github.com/singer-io/tap-stripe/blob/1223197217ab7c8a42c96d39474750bccf838dfd/tests/test_all_fields.py#L122

Reply:
removed from this dict